.circleci/scripts/deps-install.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+set -x
+
+yarn --frozen-lockfile --har
```

Comment:
```suggestion
yarn --frozen-lockfile --ignore-scripts --har
```
I'm guessing this is what was meant? Otherwise it's probably not especially useful to run the build steps separately, as they'd have been run once already.

Reply:
oh lul